packages/nuxi/src/commands/dev.ts

```diff
@@ -92,7 +92,7 @@ const command = defineCommand({
     // Prepare
     overrideEnv('development')
     const cwd = resolve(ctx.args.cwd || ctx.args.rootDir)
-    showVersions(cwd)
+    await showVersions(cwd)
```

Comment:
Do we need to await it here? I wonder if it matters order-wise and would improve dev server startup time / build speed (for `build.ts`) if we would not await it (so it can run in parallel)

Reply:
Yes, you're right 🙂 removed